src/test/java/org/openrewrite/staticanalysis/RenameLocalVariablesToCamelCaseTest.java

```diff
@@ -114,6 +116,7 @@ public int addTen(int renameOne) {
     }
 
     @SuppressWarnings("JavadocDeclaration")
+    @Disabled
```

Comment:
@joanvr did you notice this? I've disabled this test since it fixed an issue that might be important if we ever decide to start (limited) support for renaming method args.

Reply:
Maybe we could try to rename param references in annotations too? Although I'm not sure if we can have a generic enough approach

Reply:
Maybe, but even if there is 1 annotation, it does not mean that it's always used only with that annotation. It's a slippery slope and there will likely still be cases where we could break something.